src/planscape/planning/serializers.py

```diff
@@ -52,6 +52,7 @@ class Meta:
 
 
 class ConfigurationSerializer(serializers.Serializer):
+    id = serializers.IntegerField()
```

Comment:
can you make this question_id instead to make it extra clear?
Also, can you add this not to be required / or nullable? If not listing scenarios that do not have this will fail. Unless we are cool with that (we'll need to nuke the db entries)